<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowUi</class>
 <widget class="QDialog" name="MainWindowUi">
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Combinations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="gbTableCards">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Table cards</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbPlayerCards">
       <property name="minimumSize">
        <size>
         <width>176</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>176</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Player cards</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbGameTypes">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Game types</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbTexasHoldem">
          <property name="text">
           <string>Texas Holdem</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgGameTypes</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbOmaha">
          <property name="text">
           <string>Omaha</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgGameTypes</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbActions">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Actions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="btnApply">
          <property name="text">
           <string>apply</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnCancel">
            <property name="text">
             <string>cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLog">
            <property name="text">
             <string>in log</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lblCardsCombination">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="ltDeck"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bgGameTypes"/>
 </buttongroups>
</ui>
